extensions/wikia/Email/EmailController.class.php

```diff
@@ -644,4 +644,13 @@ private static function getLegendName() {
 
 		return $legendName;
 	}
+
+	/**
+	 * A wrapper for wfMessage() which removes possibility of database lookup for message text
```

Comment:
Can we update this comment to something more like `"A wrapper for wfMessage() which removes the possibility of the message being overridden in the MediaWiki namespace."`? That's what the intent of this method is, database lookup is a little vague for what we're actually doing.